source/catalog.html

```diff
@@ -0,0 +1,246 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <link rel="icon" type="image/png" sizes="32x32" href="img/icon/favicon.png">
+    <link rel="stylesheet" href="css/">
+    <title>HTML Academy: Кэт энерджи</title>
+  </head>
+  <body>
+    <h1 class="visually hidden" lang="en">CAT ENERGY</h1>
+    <header class="header">
+      <a class="header__logo logo" href="#">
+        <img class="logo__img" width="33" height="37" alt="Наш маскот." src="img/icon/logo_mobile.svg">
+      </a>
+      <a class="header__logo-main logo-main" href="#">
+        <img class="logo-main__img" width="101" height="18" alt="Логотип «Кэт Энерджи»." src="img/icon/logo_main.svg">
+      </a>
+      <button class="header__toggle" type="button" aria-expanded="false" aria-controls="main-nav">
+        Открыть меню
+      </button>
+      <nav class="header__popover menu" id="main-nav">
+        <ul class="menu__content-list">
+          <li class="menu__content-item">
+            <a class="menu__content-link" href="index.html">Главная</a>
+           </li>
+          <li class="menu__content-item">
+            <a class="menu__content-link">Каталог продукции</a>
+          </li>
+          <li class="menu__content-item">
+            <a class="menu__content-link" href="form.html">Подбор программы</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section class="catalog">
+        <h2 class="catalog__title">Каталог продукции</h2>
+        <div class="catalog__wrapper">
+          <ul class="catalog__list">
+            <li class="catalog__card card">
+              <a class="card__link-img" href="#">
+                <img class="card__img" width="68" height="86" alt="Корм с курицей 500г." src="img/catalog/chicken_small.png">
+              </a>
+              <a class="card__link-title" href="#">Cat Energy PRO 500 г</a>
```

Comment:
Можно добавить заголовок
```suggestion
              <a class="card__link-title" href="#">
                <h3>Cat Energy PRO 500 г</h3>
              </a>
```